vendor/dummy-module.js

```diff
@@ -0,0 +1 @@
+export default {};
```

Comment:
this project does not have a dependency on `ember-cli-babel` because I wanted to keep the dependency tree for non-Ember users small. will this still be transpiled correctly without the `ember-cli-babel` dependency?

Reply:
looking at the CI failure, I guess the answer is: nope

Reply:
Oops. I forgot to test this locally *without* embroider.

I think I see a supported path, trying something else.